containers/requirements.txt

```diff
@@ -0,0 +1,32 @@
+cachetools==5.3.1
```

Comment:
you could also use pip-tools < https://www.youtube.com/watch?v=G8PApVvdkjQ

Reply:
nice! Thanks!